data/human/kestrel.txt

```diff
@@ -17,7 +17,14 @@ mission "Kestrel Testing"
 	source "Wayfarer"
 	waypoint "Umbral"
 	to offer
-		"combat rating" > 8000
+		or
+			"combat rating" > 6000
+			and
+				"combat rating" > 2500
+				has "main plot completed"
+			and
+				"combat rating" > 1500
+				has "global: unlocked kestrel"
```

Comment:
OK, I can see what you mean. When you first play the game, I think you should not get the Kestrel before you finish the Free Worlds campaign unless you work for it, so Dreadnoughts won't be obsolete. If you finish the Free Worlds campaign, your notoriety from fighting the Pug should be enough. That certainly counts as "exploits in battle." Once you get the Kestrel a few times, you should unlock it at the start of the game as now you know about aliens and the Kestrel shouldn't be made obsolete by alien ships. Considering the fourth wall nature of the mission, I think this is fitting mentioning your "exploits" from earlier games. The following suggestion is a one-liner, gets you the Kestrel after 2000 combat rating after 4 games, guarantees the Kestrel after the Free Worlds campaign, and eventually your past exploits from other games are enough to get it immediately upon creating a new pilot, which your one-liner also does.
```suggestion
                "combat rating" > 8000 - "main plot completed" * 8000 - "global: unlocked kestrel" * 1500
```